### PR TITLE
Add summary counts of for oai to s3

### PR DESCRIPTION
**Pull request by bibliotechy**
Adds summary logging for oai to s3 in the case where many sets have been
harvested.

Also adds return values to the python method so the summary counts will
be stored in XCOM.

### CODE REVIEW

File: tests/test_harvest.py [hunk 1]
```diff
@@ -455,3 +456,40 @@ def test_oai_to_s3_harvest(self, mock_harvest, mock_prefix, mock_process, **kwar
         self.assertTrue(mock_harvest.called)
         self.assertTrue(mock_prefix.called)
         self.assertTrue(mock_process.called)
+
+
+    @mock.patch("tulflow.harvest.harvest_oai")
```
Comment (sensei100) [inline tests/test_harvest.py]: Should lines 461-463 be indented?

File: tests/test_harvest.py [hunk 2]
```diff
@@ -455,3 +456,40 @@ def test_oai_to_s3_harvest(self, mock_harvest, mock_prefix, mock_process, **kwar
...
+        kwargs["until"] = "until"
+        kwargs["dag"] = dag
+        kwargs["timestamp"] = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+        mock_process.return_value = {"updated": 2, "deleted": 0 }
+        response = harvest.oai_to_s3(**kwargs)
+        self.assertEqual(response, {"updated": 4, "deleted": 0 })
+
+    @mock.patch("tulflow.harvest.harvest_oai")
+    @mock.patch("tulflow.harvest.dag_s3_prefix")
+    @mock.patch("tulflow.harvest.process_xml")
```
Comment (sensei100) [inline tests/test_harvest.py]: Same with 487-480?

File: tests/test_harvest.py (hunk 1 above)
Comment (bibliotechy) [reply tests/test_harvest.py]: @sensei100 Nope, those are python decorators that need to be lined up with the function definition, as per the examples in https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch

File: tests/test_harvest.py (hunk 2 above)
Comment (bibliotechy) [reply tests/test_harvest.py]: Same as above

**Comment by bibliotechy** [comment]
@sensei100 
From the Airflow docs (https://airflow.apache.org/docs/stable/concepts.html?#concepts-xcom)

> In addition, if a task returns a value (either from its Operator’s execute() method, or from a PythonOperator’s python_callable function), then an XCom containing that value is automatically pushed.

**Pull request merged**